wordpress-mobile/gutenberg-mobile
# Fix typography breakage in master

To a version where the typography panel is not added to block settings.

GB PR: https://github.com/WordPress/gutenberg/pull/21527

To test:
 - Start the demo app
 - Select a paragraph block
 - Check that no block settings icon shows up

PR submission checklist:
